scripts/migrate/export-legacy-papers.py

```diff
@@ -0,0 +1,159 @@
+            if DEBUG: print "PDF: " + pdf
+
+            tauthor = None
+            authorNames = []
+            authorEmails = []
+            if DEBUG: print("\t\tAuthors: " + authors['name'])
+            for sub in authors['subscribers']:
+                user = legacyUsers.find_one({"_id" : sub})
+                if DEBUG: print '\t\t\t' + user['fullname']
+                authorNames.append('"' +  user['fullname'] + '"')
+                # take the first email for this author (they can specify multiple emails)
+                # We may want to check that it's verified or the preferred email
+                firstEmail = next(iter(user['emails'] or []), None)
+                authorEmails.append('"' + str(firstEmail['email']) + '"')
+                # assume the first person listed is the "true author"
+                if tauthor is None:
+                    tauthor = firstEmail['email']
+
+            authorEmailList = ', '.join(authorEmails)
+            authorNameList = ', '.join(authorNames)
+
+            dttm = doc['created'];
+            t =  datetime.datetime(dttm.year, dttm.month, dttm.day, dttm.hour, dttm.minute)
+            createddt = str(time.mktime(t.timetuple()) * 1000)
+
+            forumid = shortid.ShortId().generate()
+
+            # note: for now, we'll just use count for the paper number, normally this
+            # would come from the "counter" collection in Arango
+
+            # rather than replacing double quotes with '\"', we'll replace them with a
+            # single quote. There is some data that the escaped double quote wouldn't work for
+            # such as 'na\"ive'. So rather than handle all special cases, just use a single quote.
+            abstract = doc['summary'].replace('"', '\'')
+            # the import process to Arango wants one JSON record per line.
+            noteJSON = ('{'
+                '"tddate": null,'
+                '"replyto": null,'
+                '"active": true,'
+                '"tmdate": ' + createddt + ','
+                '"tcdate": ' + createddt + ','
+                '"number": ' + str(count) + ','
+                '"id": "' + forumid + '",'
+                '"invitation": "' + submitInvite + '",'
+                '"forum":   "' + forumid + '",'
+                '"tauthor":  "' + tauthor + '",'
+                '"signatures": [],'
+                '"readers": ["everyone"],'
+                '"writers": [],'
+                '"content": {'
+                '"decision" : "' + endorsement + '",'
+                '"title": "' + doc['title'] + '",'
+                    '"abstract": "' + abstract + '",'
+                    '"pdf":  "' + pdf + '",'
+                    '"conflicts": [],'
+                    '"authors":   [' + authorNameList + '],'
+                    '"keywords": [],'
+                    '"authorids":   [' + authorEmailList + ']'
+                '},'
+                '"legacy_migration" : true'
```

Comment:
I think we don't need to save this property, we need to keep the note with the same schema than the other papers.